app/models/wallet_user.model.ts

```diff
@@ -0,0 +1,26 @@
+import sequelize from '../../config/sequelize';
+import * as Sequelize from 'sequelize';
+
+const WalletAddress = sequelize.define('tb_wallets_users', {
+  user_id: {
+    type: Sequelize.STRING(11),
+    allowNull: false
+  },
+  client_id: {
```

Comment:
I don't think we need `client_id` here, don't you think its redundant here? Given that wallet has client_id already